app/src/main/java/com/example/musicapp/dto/Instrument.kt

```diff
@@ -1,6 +1,7 @@
 package com.example.musicapp.dto
 
 data class Instrument(var id: Int, var name: String, var type: String) {
+    //Represents an instrument
```

Comment:
Additional comment to help make it easier for others to follow

Reply:
Inline comments tend to be avoided by developers.  Code should be self-documenting, and inline comments are often seen as an indication that the code is not self-documenting.  If it is self-documenting, then inline comments are seen as redudant.